src/commands/config/index.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  ChannelType,
+  ChatInputCommandInteraction,
+  GuildTextBasedChannel
+} from 'discord.js';
+import { PermissionFlagsBits } from 'discord.js';
+
+import { Command } from '../../types/command.js';
+import type {
+  CommandBuilder,
+  CommandBuilderOutput
+} from '../../types/command-builder.js';
+import { enableRandomVoiceChannelNamesConfig } from './enable-random-voice-channel-names-config.js';
+import { enableKarmaTrackingConfig } from './enable-karma-tracking-config.js';
+
+enum ConfigSubcommand {
+  EnableKarmaTracking = 'enable-karma-tracking',
+  EnableRandomVoiceChannels = 'enable-random-voice-channels'
+}
+
+// TODO: Not implemented subcommands:
+// setup: Once we have more than one config for a guild, we can use setup as a wizard to guide admins through them
+// scan-members: If/when Rusty gets a website, this command will iterate through all existing users to make sure they have a document in firestore
+
+export class ConfigCommand extends Command {
+  public readonly name = 'config';
+  public readonly description = 'Configuration options';
```

Comment:
lets make this more verbose, and include the word "setting(s)" in some way so that it is easier to find.